ahn/training/_decoder.py

```diff
@@ -1,11 +1,13 @@
-import json
+import jsonlines
```

Comment:
Is it a lot of differ with using json lib?

Reply:
Not so much different I think... just for better code interface